shared/src/msi-installer/Config.wxi

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <Include>
-  <?define BaseProductName = ".NET Tracer" ?>
+  <?define BaseProductName = ".NET Monitoring" ?>
```

Comment:
Not a blocker for now, just leaving a note that we should determine the correct product name before this installer goes GA.